### ISSUE TITLE
Installing problem

### ISSUE THREAD

**Issue by wilcar** [vosonlab/SocialMediaLab#17]
I can't install SocialMediaLab package on my compter (Rstudio)
I am using the _Absolute Beginner’s Guide to the SocialMediaLab package in R_ method page 2.

```
`install.packages(httr)`
`set_config(use_proxy(url='cache.ha.univ-nantes.fr',3128)) # working behind a proxy`
`# we have to install an earlier version of 'httr' package`
`# due to incompatibility problems with the 'twitteR' package`
`devtools::install_version("httr", version="0.6.0", repos="http://cran.us.r-project.org")`
`devtools::install_github("voson-lab/SocialMediaLab/SocialMediaLab")`
`require(SocialMediaLab)`
`if (!"magrittr" %in% installed.packages()) install.packages("magrittr")`
`require(magrittr)`
```

I have an error:

```
`lazy-load database 'D:/Documents/Cariou-w/Documents/R/win-library/3.2/httr/R/httr.rdb' is corrupt`
 `in addition: Warning messages:`
`1: le package ‘SocialMediaLab’ a été compilé avec la version R 3.2.5`
 `2: In get(Info[i, 1], envir = env) : internal error -3 in R_decompress1`
 `> if (!"magrittr" %in% installed.packages()) install.packages("magrittr")`
 `> require(magrittr)`
```

**Comment by rjackland** [comment]
Hi,

Here are few suggestions that may help.

First, we have updated the CRAN version to 0.23.0 and so that is now in line with the github (development) version.  So I would recommend that you just install the CRAN version using: install.packages("SocialMediaLab")

Second, in the "Absolute Beginners" guide there is a workaround for problems relating to the "httr" pacakge - we install an earlier version of httr.  But in recent use of SocialMediaLab, we've found no problems with httr, so I recommend you just go with the latest (CRAN) version of httr.  If it is not already installed on your system, then it should be automatically installed as a dependency of SocialMediaLab.

Finally, I wanted to point out that even though I suggest above there is no need to install the github version of SocialMediaLab (because CRAN version is up-to-date), just a heads up: we have recently changed the github repository name, so the correct syntax for installing SocialMediaLab from github is:

devtools::install_github("vosonlab/SocialMediaLab/SocialMediaLab")

Regards

**Comment by wilfridcariou** [comment]
Hi,  I tried your first solution without success

     install.packages ("httr", dependencies = TRUE)
     library (httr)
     install.packages("SocialMediaLab", dependencies = TRUE)
     library (SocialMediaLab)

Unfortunately I still have a message error (in french)

    Error: le chargement du package ou de l'espace de noms a échoué pour ‘SocialMediaLab’

 translation:
`package or namespace load failed`


my session info (may help)


     sessionInfo() 

R version 3.2.3 (2015-12-10) Platform: x86_64-w64-mingw32/x64 (64-bit) Running  under: Windows >= 8 x64 (build 9200) locale: [1] LC_COLLATE=French_France.1252  
LC_CTYPE=French_France.1252 LC_MONETARY=French_France.1252 LC_NUMERIC=C [5] LC_TIME=French_France.1252 

attached base packages: [1] stats graphics grDevices utils datasets methods base
other attached packages: [1] httr_1.2.1 
loaded via a namespace (and not attached): [1] rjson_0.2.15 R6_2.1.3 parallel_3.2.3 tools_3.2.3 NLP_0.1-9 RCurl_1.95-4.8 yaml_2.1.13 Rcpp_0.12.6 [9] data.table_1.9.6 knitr_1.14 httpuv_1.3.3 chron_2.3-47 bitops_1.0-6


Thank you for help!

**Comment by timothyjgraham** [comment]
Hi @wilfridcariou - did you have any luck solving this issue?

**Comment by jgaf151** [comment]
I had similar issue not sure if it was entirely the same. My problem was that the package was not installing all the dependencies, even if you force R to do it. Had to install NLP, slam, base64, bit64, and DBI individually. Hope this helps.

**Comment by EricPeter** [comment]
package ‘SocialMediaLab’ is not available (for R version 3.4.4)
 I have a problem with my r studio as shown above

**Comment by vosonlab** [comment]
We have updated SocialMediaLab and the github version (0.23.3) is now fully compatible with the latest version of R.

However as of 5 July 2018 we have **ceased development of SocialMediaLab** and have **created a replacement R package** [vosonSML](https://github.com/vosonlab/vosonSML), which is available on CRAN.  

vosonSML (version 0.23.4) on CRAN is identical to SocialMediaLab (version 0.23.3) on github.  It is fully compatible with the latest version of R and so it should install in R Studio.
